core/IO.carp

```diff
@@ -1,14 +1,78 @@
+; TODO: check if documentation needs parameters listed or if they are in function signature
```

Comment:
The parameters in the documentation are listed. If you mention them, I suggest you use Markdown syntax (i.e. backticks) to refer to them!

Reply:
What confused me a bit was: Parameters seem to be listed only for functions, not for templates or external.
So I'll update the doc strings for fopen and fread but not for fseek.

Reply:
This is true, since templates and external functions don’t have argument names that are visible to Carp.